pages/en/lb3/readmes/loopback-connector-openapi.md

```diff
@@ -0,0 +1,272 @@
+# loopback-connector-openapi
```

Comment:
I think this file is copied using the script https://github.com/strongloop/loopback.io/blob/gh-pages/update-readmes.sh.